cg/store/crud/update.py

```diff
@@ -21,12 +21,10 @@ def __init__(self, session: Session):
         super().__init__(session=session)
         self.session = session
 
-    def update_sample_comment(self, sample: Sample, comment: str) -> None:
+    def update_sample_comment(self, sample_id: int, comment: str) -> None:
         """Update comment on sample with the provided comment."""
-        if sample.comment:
-            sample.comment = sample.comment + " " + comment
-        else:
-            sample.comment = comment
+        sample: Sample | None = self.get_sample_by_entry_id(sample_id)
+        sample.comment = f"{sample.comment} {comment}" if sample.comment else comment
```

Comment:
Do we need a null check here?

Reply:
Fixed in https://github.com/Clinical-Genomics/cg/pull/3507/commits/70e66ba5e2c0878722aa412686340482fe7f9c4f